model/client4.go

```diff
@@ -3090,9 +3090,11 @@ func (c *Client4) CreateEmoji(emoji *Emoji, image []byte, filename string) (*Emo
 	body := &bytes.Buffer{}
 	writer := multipart.NewWriter(body)
 
-	if part, err := writer.CreateFormFile("image", filename); err != nil {
+	part, err := writer.CreateFormFile("image", filename)
```

Comment:
The changes in this file seem unnecessary? Can we revert for clarity?

Reply:
yes , revert it

Reply:
If these changes are still unnecessary, can you go ahead and revert as such?